debian/latest/Dockerfile

```diff
@@ -0,0 +1,68 @@
+###############################################################
+#
+# Copyright (C) 2019 Jamese Fuller <jim.fuller@webcomposite.com>
+#
+# SPDX-License-Identifier: MIT
+
+###############################################################
+
+FROM debian:latest AS builder
```

Comment:
Having a tag here would allow for reproducible builds, instead of using a `:latest` which can change under your feet.